spec/services/builders/review_turnaround_spec.rb

```diff
@@ -7,26 +7,58 @@
     let(:santib) { create(:user, login: 'santib') }
     let(:pr) { create(:pull_request, owner: vita, project: project) }
 
-    let(:rr) do
-      create(:review_request, owner: vita, reviewer: santib, project: project, pull_request: pr)
+    before do
+      allow_any_instance_of(Events::Review).to receive(:build_review_turnaround)
```

Comment:
Wouldn't `Events::Review.skip_callback(:create, :after, :build_review_turnaround)` work? It seems more _intention revealing_

Reply:
the `skip_callback` is dangerous 😆 , when i used it i didn't know that method works across the tests i mean, it will skip all the callbacks for everytime that a event creation is invoked, that's why i changed by a mock

Reply:
Oh, I didn't know that, good call!

Reply:
If you asked me, these are the things why I don't like these hooks, and I'd rather do without them.
But no, you didn't ask me 😂

Reply:
i totally agree with you,  sometimes it is really confusing 😆